src/index.js

```diff
@@ -26,3 +27,4 @@ ReactDOM.render(
     document.getElementById('root')
 );
 
+register();
```

Comment:
Can this function in `serviceWorkerRegistration.js` be renamed to something that specifies this is for the service-worker? Right now, there is this call to `register()` in index.js but it's unclear what it does. Maybe something like `registerServiceWorker()`?